R/dp_commit.R

```diff
@@ -58,3 +55,22 @@ dp_commit <- function(project_path = fs::path_wd(),
 
   return(repo)
 }
+
+get_data_object_path <- function(project_path) {
```

Comment:
There should be no issue if the data product was initially generated in one format and later switched to another, provided that the repository is freshly cloned or synchronized, therefore the contents of the output_files directory are ignored. However, it could be problematic if a user continues to develop the data product within an existing repository and decides to switch the format type. While this scenario is unlikely, it would be prudent to include a warning in the vignette explaining how the new type parameter functions.